src/main/java/com/zombiecastlerush/building/Room.java

```diff
@@ -9,9 +9,10 @@
     private String description;
     public Inventory inventory = new Inventory();
```

Comment:
same question. why we need a public inventory?

Reply:
will address in later PR